### DISCUSSION TITLE
umi4 在打包的时候对于public目录进行minify操作，降低打包速度而且报错了。

### DISCUSSION THREAD

**Discussion by bluesaiz** [umijs/umi#11468] [Q&A]
在umi3的时候，打包正常，升级到umi4的时候，打包报错了。
 
![image](https://github.com/umijs/umi/assets/40681310/d8e1c365-af34-41a7-ac7d-a28d7779364c)

请问能在打包的时候排除对于public文件夹的minify操作的吗？public里面的文件已经处理过了。

**Comment by fz6m** [comment]
umi 4 不会压缩 `public` 里的文件，应该是其他的压缩问题，尝试调整下 [`jsMinifier`](https://umijs.org/docs/api/config#jsminifier-webpack) 看看吧。

如还有问题，需要提供一个最小复现。
